app/frontend/wallet/shelley/helpers/poolCertificateUtils.ts

```diff
@@ -167,33 +167,33 @@ const parseStakepoolRelays = (relays: _PoolRelay[]): TxStakepoolRelay[] =>
           type: TxRelayType.SINGLE_HOST_IP,
           params: {
             portNumber: relay.portNumber,
-            ipv4: relay.ipv4 ? ipv4BufToAddress(relay.ipv4) : null,
-            ipv6: relay.ipv6 ? ipv6BufToAddress(relay.ipv6) : null,
+            ipv4: relay.ipv4 ? ipv4BufToAddress(relay.ipv4) : undefined,
+            ipv6: relay.ipv6 ? ipv6BufToAddress(relay.ipv6) : undefined,
           },
         }
       case TxRelayTypes.SINGLE_HOST_NAME:
         return {
           type: TxRelayType.SINGLE_HOST_NAME,
           params: {
             portNumber: relay.portNumber,
-            dnsName: relay.dnsName,
+            dnsName: relay.dnsName || '',
```

Comment:
this fallback to empty string seems really fragile as it is apparently a required property as per the type signature. A runtime assertion on the property being defined would be more sensible, same below